setup.py

```diff
@@ -26,7 +26,7 @@ def local_scheme(version):
     package_dir={"": "src"},
     use_scm_version={"local_scheme": local_scheme},
     setup_requires=["setuptools_scm"],
-    install_requires=["setuptools", "wcwidth"],
+    install_requires=["setuptools", "wcwidth", "colorama"],
```

Comment:
We'd probably want to put this as an extra (see `tests` in `extras_require` below) so it's only installed when people actually want to use, but we can deal with that later.

Reply:
Fixing